README.md

```diff
@@ -1,16 +1,71 @@
-<p align="center">
-   <img src="https://github.com/xetdata/pyxet/blob/0c7608c97f6a2a0cb2c83dd38fb717913c4d7522/docs/images/logo.png" alt="logo" width="400" />
-</p>
+# Git-Xet: Scale Git to Terabyte-sized Repos
 
-## What is it?
-xet-core is a collection of Rust crates which provide the low level, high performance, functionality and API bindings for [pyxet](https://pyxet.readthedocs.io/en/latest/) and [git-xet](https://xethub.com/assets/docs/getting-started/install). 
+Git Xet is a plugin for git that allows git repositories to scale to terabytes of data.  
 
-For more details, see [Git is for Data](https://www.cidrdb.org/cidr2023/papers/p43-low.pdf).
+## Design Goals
+
+### Binary Content Deduplication
+
+Binary content is deduplicated across commits, files, and blocks of identical content in otherwise different files.   This allows git to efficiently work with repositories containing large datasets that evolve over time.  See our paper at CIDR 2023, [Git is for Data](https://www.cidrdb.org/cidr2023/papers/p43-low.pdf), for more information on the technical details of this process.  
```

Comment:
It might be better to link to our blog post about the paper as it includes a little more context and the abstract which may be more consumable by more people. It does link to the paper too. https://about.xethub.com/blog/git-is-for-data-published-in-cidr-2023